main.js

```diff
@@ -0,0 +1,286 @@
+const gameBoard = (() => {
+
+    let playerOneSelections = [];
+    let playerTwoSelections = [];
+
+    function resetPlayers() {
+        this.playerOneSelections = [];
+        this.playerTwoSelections = [];
+    };
+
+    return {
+        playerOneSelections,
+        playerTwoSelections,
+        resetPlayers,
+    };
+})();
+
+
+
+/////////////////////////////////////////////
+
+
+
+
+const Player = (name, mark) => {
+
+    
+    return {
+        name,
+        mark,
+    };
+};
+
+const player1 = Player('Player 1', 'X');
+const player2 = Player('Player 2', 'O');
+
+
+
+//////////////////////////////////////
+
+
+
+const game = (() => {
+    
+    const playerTurn = document.getElementById('player-turn');
+    playerTurn.style.display = "none";
+    let x_turn = true;
+    let roundCount = 0;
+    let player1name;
+    let player2name;
```

Comment:
You could let these = 'X' and 'O' here, so that later on if the user doesn't input their name, it would default back to showing Player X turn or Player O turn rather then having it be: " 's turn" if the user doesn't input a name